### ISSUE TITLE
Update / restyle for Preferences > Payments

### ISSUE THREAD

**Issue by bsclifton** [brave/browser-laptop#5494]
Starting off with a picture (courtesy of @bradleyrichter)
![image](https://cloud.githubusercontent.com/assets/13509546/20120076/18af17f0-a5c0-11e6-987a-f4a3e6019b49.png)

Style updates include:
- sortableTable no longer hides it's style (should properly show the column headers)
- re-connected styles for all widgets with master styles

New functionality:
- "hide excluded sites" toggle
- exclude button on the left (in addition to the existing right click one)
(for blacklisting sites that you don't want to see regardless of their include/exclude state)

**Comment by srirambv** [comment]
Some how **Thanks for helping support your favorite website!** doesn't sound right.

Instead can it be **Thanks for your help in supporting your favorite website!** 

@bsclifton @bradleyrichter Thoughts ?

**Comment by bradleyrichter** [comment]
@srirambv How about the above string? 

"Thanks for supporting your favorite websites!"

**Comment by srirambv** [comment]
Yes that sounds better 👍

**Comment by bsclifton** [comment]
@bradleyrichter anything else needed before we can start this one?

**Comment by bradleyrichter** [comment]
This is good to go until we add the ability to add an individual site.

**Comment by bradleyrichter** [comment]
@mrose17 @jkup 

This is the task I had mentioned today...

**Comment by jkup** [comment]
yep it's in 0.12.11!

**Comment by jkup** [comment]
@mrose17 Is there an endpoint in ledger-client for blacklisting a site?

**Comment by mrose17** [comment]
there is no concept of "black-listing". there is a concept of an exclusion list. there are two, each browser has a private exclusion list (right click on a site on about:preferences#payments and select "Never include this site". there is also a global exclusion list, that is under review.

**Comment by jkup** [comment]
@mrose17 Ok this ticket looks to be calling for a feature that doesn't exist then? + @bsclifton

**Comment by mrose17** [comment]
i think you should DM @bsclifton and @bradleyrichter to see what they are looking for...

**Comment by bradleyrichter** [comment]
The "blacklist" column on the left is simply a more-accessible shortcut to the already existing context-menu:

![image](https://cloud.githubusercontent.com/assets/13509546/20643080/a47caeca-b3d3-11e6-98e9-39b937d98b5a.png)

This function is used for sites in which a user does not want to ever pay, or perhaps even see in their list. One example would be for a site that we have not added to our global exclude list yet but the user spend time there daily.

Youtube Red might be a good example. 

Or an offensive click-bait site that was visited using a shared computer and user-A prefers that user-B does not see that the site was visited.

**Comment by bsclifton** [comment]
@jkup this would be a new feature :smile: There is something similar (but it's not done yet) which you can check out in `js/about/history.js`.  There, you'll see a quick "delete" button on each row (using the sortableTable control)

**Comment by jkup** [comment]
@bsclifton ah ok cool.. how should I implement this? Do we add some global state for blacklistedSites or something like that? And then just check against that when we display the ledger table?

**Comment by bsclifton** [comment]
@jkup I think showing the [FontAwesome Ban icon](http://fontawesome.io/icon/ban/) on hover and then executing the same code that is executed when picking "Never include this site" would do the trick :smile: No new functionality needed per-se, since you can already right click one by one, but offering a button as the first column will make it very easy to process your list

**Comment by jkup** [comment]
Ohhhhhh ok ok ok ok got it!

**Comment by bradleyrichter** [comment]
This "quick remove" column came from the original history makeover design. 

Ideally we can use it there as well. It should work in a similar way to the the search engine prefs default column except that the hover state also shows the "no" symbol and clicking removes it from the list.

**Comment by luixxiul** [comment]
`.titleBar` and `.walletBar` are almost ready:

<img width="850" alt="screenshot 2016-12-03 23 31 57" src="https://cloud.githubusercontent.com/assets/3362943/20859971/df5ef268-b9b0-11e6-952e-15b2e509dde4.png">

**Comment by bsclifton** [comment]
Issue is resolved with https://github.com/brave/browser-laptop/pull/6101, which is ready for merge (after 0.13.0 ships). Milestone is already set and is correct.

**Comment by bsclifton** [comment]
Fixed with https://github.com/brave/browser-laptop/pull/6101 which was merged into branch `0.13.1-branch`.
